lib/Froxlor/Api/Commands/SubDomains.php

```diff
@@ -562,7 +572,7 @@ public function update()
 		if ($result['parentdomainid'] != '0' && ($result['subcanemaildomain'] == '1' || $result['subcanemaildomain'] == '2') && $isemaildomain != $result['isemaildomain']) {
 			$isemaildomain = intval($isemaildomain);
 		} elseif ($result['parentdomainid'] != '0') {
-			$isemaildomain = $result['subcanemaildomain'] == '3' ? 1 : 0;
+//			$isemaildomain = $result['subcanemaildomain'] == '3' ? 1 : 0; DF8OE
```

Comment:
Was this done for testing purposes? Or what did you mean by " workaround for not destroying all emailaccounts when toggling is email domain"?

Please do not just comment out things that could possibly lead to unintended behaviour